tuner/tuner/libtuner.py

```diff
@@ -696,6 +725,51 @@ def append_to_file(lines: list[str], filepath: Path, title: str = "") -> None:
         file.write("\n")
 
 
+def get_benchmark_context(benchmarks: Optional[dataclass]) -> dict:
+    """Gathers system information for the context section of the JSON output."""
+    cpu_info = psutil.cpu_freq()
+    num_cpus = psutil.cpu_count(logical=True)
+
+    benchmark_times = [benchmark.benchmark_time_in_seconds for benchmark in benchmarks]
+
+    if benchmark_times:
+        mean_time = sum(benchmark_times) / len(benchmark_times)
+    else:
+        mean_time = None
```

Comment:
we can simplify this a bit:
```suggestion
    mean_time = None
    if benchmark_times:
        mean_time = sum(benchmark_times) / len(benchmark_times)
```